cmd/server-main.go

```diff
@@ -482,8 +483,12 @@ func serverMain(c *cli.Context) {
 		fatalIf(errors.New("empty directory"), "Configuration directory cannot be empty.")
 	}
 
+	// Disallow relative paths, figure out absolute paths.
+	configDirAbs, err := filepath.Abs(configDir)
```

Comment:
Upon error `configDir` gets replaced when printing.. to empty string. and what do you mean "block" it?